composer.json

```diff
@@ -25,7 +25,7 @@
         "doctrine/dbal": "^2.9.3",
         "doctrine/event-manager": "^1.1",
         "doctrine/instantiator": "^1.3",
-        "doctrine/persistence": "^1.2",
+        "doctrine/persistence": "^1.3.3",
```

Comment:
```suggestion
        "doctrine/persistence": "^1.4 || ^2.0",
```

Reply:
I'd do this in a separate PR, once 1.4 is effectively published, cause I'm not going to get a green build otherwise, am I?

Reply:
Um, yeah. Absolutely.